cache.h

```diff
@@ -92,185 +70,209 @@ class NoiseField {
 public:
     float temperature;
     float humidity;
-    float ocean;
+    bool ocean;

```

Comment:
> Why is this changing to a boolean?

This is changing to a boolean because given a point on the terrain, we want to know if there's water there or not.

> How do the ocean and river values relate?

Good question. ATM the only noise for water is the `oceanNoise` ( and river noise was used before but not now ).
So I'll do some cleanup to make sure we don't have a river noise ( At least for now ! )

> Would be great to have this documented in notion

Good point. I think we'll need to document a lot of the procgen systems we're using once we decide something is working for us and we want to keep it...

Reply:
👍